common/wireless.c

```diff
@@ -84,198 +84,200 @@ static void wirelessSendPacket(bool cpy);
-    "   st Z,__tmp_reg__      \n"
-    "   adiw r28,1            \n"   // Y++
-    "   adiw r30,1            \n"   // Z++
-    "   subi r26,1            \n"
-    "   brcc roll_loop        \n"   // 8 times loop
-    "   sbiw r28,8            \n"   // Y-=8
-    "   ret "
-);
+	"left_roll:               \n"
+	"   ldd r26,Y+7           \n"
+	"   lsl r26               \n"
+	"   in r27,__SREG__       \n"   // save carry
+	"   ldi r26,7             \n"   // 8 times
+	"roll_loop:               \n"
+	"   ld __tmp_reg__,Y      \n"
+	"   out __SREG__,r27      \n"   // restore carry
+	"   rol __tmp_reg__       \n"
+	"   in r27,__SREG__       \n"   // save carry
+	"   st Z,__tmp_reg__      \n"
+	"   adiw r28,1            \n"   // Y++
+	"   adiw r30,1            \n"   // Z++
+	"   subi r26,1            \n"
+	"   brcc roll_loop        \n"   // 8 times loop
+	"   sbiw r28,8            \n"   // Y-=8
+	"   ret "
+	);
 
 /*!
  *******************************************************************************
  *  encrypt / decrypt
- *  \note symetric operation 
+ *  \note symetric operation
  ******************************************************************************/
-
-
-static void encrypt_decrypt (uint8_t* p, uint8_t len) {
-    uint8_t i=0;
-    uint8_t buf[8];
-    while(i<len) {
-        xtea_enc(buf,&RTC,K_enc);
-        RTC.pkt_cnt++;
-        do {
-            p[i]^=buf[i&7];
-            i++;
-            if (i>=len) return; //done
-        } while ((i&7)!=0);
-    }
+static void encrypt_decrypt(uint8_t *p, uint8_t len)
+{
+	uint8_t i = 0;
+	uint8_t buf[8];
+
+	while (i < len) {
+		xtea_enc(buf, &RTC, K_enc);
+		RTC.pkt_cnt++;
+		do {
+			p[i] ^= buf[i & 7];
+			i++;
+			if (i >= len) return; //done
+		} while ((i & 7) != 0);
+	}
 }
 
 /*!
  *******************************************************************************
  *  wireless send Done
  ******************************************************************************/
-void wirelessSendDone(void) {
-    RFM_INT_DIS();
-    rfm_mode    = rfmmode_stop;
-    #if defined(MASTER_CONFIG_H)
-        wireless_buf_ptr=0;
-    #endif
-    rfm_framepos=0;
-
-    RFM_FIFO_OFF();
-    RFM_FIFO_ON();
-    RFM_RX_ON();    //re-enable RX
-    rfm_mode = rfmmode_rx;
-    RFM_INT_EN(); // enable RFM interrupt
-
-    #if !defined(MASTER_CONFIG_H)
-        wirelessTimerCase = WL_TIMER_RX_TMO;
-        while (ASSR & (_BV(TCR2UB))) {;}
-        RTC_timer_set(RTC_TIMER_RFM, (uint8_t)(RTC_s256 + WLTIME_TIMEOUT));    
-        COM_print_time('r');
-    #endif    
+void wirelessSendDone(void)
+{
+	RFM_INT_DIS();
+	rfm_mode = rfmmode_stop;
+#if defined(MASTER_CONFIG_H)
+	wireless_buf_ptr = 0;
+#endif
+	rfm_framepos = 0;
+
+	RFM_FIFO_OFF();
+	RFM_FIFO_ON();
+	RFM_RX_ON();    //re-enable RX
+	rfm_mode = rfmmode_rx;
+	RFM_INT_EN();   // enable RFM interrupt
+
+#if !defined(MASTER_CONFIG_H)
+	wirelessTimerCase = WL_TIMER_RX_TMO;
+	while (ASSR & (_BV(TCR2UB))); RTC_timer_set(RTC_TIMER_RFM, (uint8_t)(RTC_s256 + WLTIME_TIMEOUT));

```

Comment:
Again, automatic formatting here failed due to empty loop body. I have seen this on many places over the code, I will not comment on all, but if we can somehow tune the "beautifier" parameters, this should be fixed. The result here is ugly.

Reply:
I'll try to fix these issues with the new uncrustify.conf in the next few days! It'll take some time to review the whole code changes..